book/jsonrpc/intro.md

```diff
@@ -125,3 +125,42 @@ cast rpc admin_nodeInfo
 cast rpc debug_traceTransaction
 cast rpc trace_replayBlockTransactions
 ```
+
+## Ports
+
+This section provides essential information about the ports used by the system, their primary purposes, and recommendations for exposure settings.
+
+### Peering Ports
+
+- **Port:** 30303
+- **Protocol:** TCP and UDP
+- **Purpose:** Peering with other nodes for synchronization of blockchain data. Nodes communicate through this port to maintain network consensus and share updated information.
+- **Exposure Recommendation:** This port should be exposed to enable seamless interaction and synchronization with other nodes in the network.
+
+### Metrics Port
+
+- **Port:** 9001
+- **Protocol:** TCP
+- **Purpose:** This port is designated for serving metrics related to the system's performance and operation. It allows internal monitoring and data collection for analysis.
+- **Exposure Recommendation:** By default, this port should not be exposed to the public. It is intended for internal monitoring and analysis purposes.
```

Comment:
metrics and peering ports do not really belong to jsonrpc section, maybe we should add a `Ports` section under `Run a Node`? @mattsse @onbjerg thoughts?

Reply:
@rkrasiuk I actually had a doubt because there were some ports that didn't belong here. But it makes sense to open a dedicated section in `run a node`. I just moved this documentation to a new section.